corehq/apps/app_manager/suite_xml.py

```diff
@@ -1907,33 +1922,26 @@ def get_target_module(case_type, module_id, with_product_details=False):
                     'action': action
                 })
             else:
-                if action.parent_tag:
-                    parent_action = form.actions.actions_meta_by_tag[action.parent_tag]['action']
-                    parent_filter = SuiteGenerator.get_parent_filter(
-                        action.parent_reference_id,
-                        parent_action.case_session_var
-                    )
+                if action.parents:
+                    for parent in action.parents:
```

Comment:
I don't think this is correct. This would load the same case multiple times by using different parent references. Although a case may have multiple indices you can only load it into the session based off one of those and not both.

So we should add some validation (not here) to make sure that there is only one parent for any `load` actions. Alternatively we should split the model so that `load` actions have a single parent and `open` actions can have multiple.